scripts/release_notes.rb

```diff
@@ -10,14 +10,19 @@
 
 URL = 'https://projects.theforeman.org'
 
-unless ARGV.size == 2
-  puts "Usage: #{$0} PROJECTS RELEASE_NAME"
+unless ARGV.size <= 2
```

Comment:
I think this isn't correct. Shouldn't it be:
```suggestion
if ARGV.size < 2
```

Reply:
Sure.